thetis/timeintegrator.py

```diff
@@ -218,6 +294,80 @@ def advance(self, t, update_forcings=None):
         for k in sorted(self.fields_old):
             self.fields_old[k].assign(self.fields[k])
 
+    def cell_residual(self, label='all'):
```

Comment:
`cell_residual` and `edge_residual` are not called anywhere. Am I missing something? They should be used in the solver, no?

Reply:
At the minute they are only used by my (external) error estimation routines, or in the callbacks